tests/sqlparser_hive.rs

```diff
@@ -566,3 +566,8 @@ fn hive() -> TestedDialects {
 fn hive_and_generic() -> TestedDialects {
     TestedDialects::new(vec![Box::new(HiveDialect {}), Box::new(GenericDialect {})])
 }
+
+#[test]
+fn parse_create_external_table_without_location() {
+    hive().verified_stmt("CREATE EXTERNAL TABLE t (c INT)");
+}
```

Comment:
Can we add this entry to [this](https://github.com/romanoff/datafusion-sqlparser-rs/blob/37581a62d3b969ac320f1dfbe6563cf441f46f30/tests/sqlparser_hive.rs#L33) existing test case instead?